tests/common.py

```diff
@@ -620,6 +621,27 @@ def mock_registry(
     return registry
 
 
+def mock_area_registry(
```

Comment:
Not sure if we want to keep this long term, but we can remove it again if not after the CI is fixed and we aren't blocking everyone

Reply:
PR #110850 used `mock_area_registry`. After the `mock_area_registry` was removed the PR was not rebased.